src/app/researcher/study/request/[orgSlug]/study-proposal.tsx

```diff
@@ -52,7 +52,7 @@ const StepperButtons: React.FC<StepperButtonsProps> = ({ form, stepIndex, isPend
     return null
 }
 
-async function uploadFile(file: File, upload: PresignedPost) {
+export async function uploadFile(file: File, upload: PresignedPost) {
```

Comment:
since these are now used by other code, lets convert  these to a 'useUploadStudyCode' hook in hooks/upload.ts or something like that.  The hook could return a function that would accept all the needed info and then upload